src/actions/guides/frontend/asset-handling.cr

```diff
@@ -125,6 +125,22 @@ class Guides::Frontend::AssetHandling < GuideAction
 
     Make sure to use the `asset` macro to get fingerprinted assets.
 
+    ## Asset host
+
+    Once your app is in production, you may want to serve up your assets through a [CDN](https://en.wikipedia.org/wiki/Content_delivery_network). To specify a differnt host, you'll use the `asset_host` option in `config/server.cr`.
```

Comment:
Typo in 'differnt'.

Also could you format these lines so they fit within 80(ish) characters? That'll make it a bit easier to comment/review in the future